src/commands/synthetics/__tests__/utils/internal.test.ts

```diff
@@ -88,7 +95,80 @@ describe('utils', () => {
       [undefined, undefined],
     ]
     test.each(cases)('toExecutionRule(%s) should return %s', (input, expectedOutput) => {
-      expect(internalUtils.toExecutionRule(input)).toEqual(expectedOutput)
+      expect(toExecutionRule(input)).toEqual(expectedOutput)
+    })
+  })
+
+  describe('overrideUtils', () => {
+    describe('parseValue', () => {
+      it('should parse boolean values correctly', () => {
+        expect(parseOverrideValue('true', 'boolean')).toBe(true)
+        expect(parseOverrideValue('True', 'boolean')).toBe(true)
+        expect(parseOverrideValue('TRUE', 'boolean')).toBe(true)
+        expect(parseOverrideValue('1', 'boolean')).toBe(true)
+        expect(parseOverrideValue('false', 'boolean')).toBe(false)
+        expect(parseOverrideValue('False', 'boolean')).toBe(false)
+        expect(parseOverrideValue('FALSE', 'boolean')).toBe(false)
+        expect(parseOverrideValue('0', 'boolean')).toBe(false)
+      })
+
+      it('should throw an error for invalid boolean values', () => {
+        expect(() => parseOverrideValue('notABoolean', 'boolean')).toThrow('Invalid boolean value: notABoolean')
+        expect(() => parseOverrideValue('', 'boolean')).toThrow('Invalid boolean value: ')
+      })
+
+      it('should parse number values correctly', () => {
+        expect(parseOverrideValue('123', 'number')).toBe(123)
+        expect(parseOverrideValue('3.14', 'number')).toBe(3.14)
+      })
+
+      it('should throw an error for invalid number values', () => {
+        expect(() => parseOverrideValue('notANumber', 'number')).toThrow('Invalid number value: notANumber')
+      })
+
+      it('should parse string values correctly', () => {
+        expect(parseOverrideValue('hello world!', 'string')).toBe('hello world!')
+      })
+
+      it('should parse enum values correctly', () => {
+        expect(parseOverrideValue('blocking', 'ExecutionRule')).toBe(ExecutionRule.BLOCKING)
+        expect(parseOverrideValue('non_blocking', 'ExecutionRule')).toBe(ExecutionRule.NON_BLOCKING)
+        expect(parseOverrideValue('skipped', 'ExecutionRule')).toBe(ExecutionRule.SKIPPED)
+      })
+      it('should throw an error for invalid enum values', () => {
+        expect(() => parseOverrideValue('invalid_enum', 'ExecutionRule')).toThrow(
+          'Invalid ExecutionRule value: invalid_enum'
+        )
+      })
+    })
```

Comment:
You're missing a case for `string[]` 🤔 i know it's not being used but the code is there already so either remove the code for now or add a test i think :D